Maven build fail

**Describe the bug**
When building maven, the following error is displayed.
```
[ERROR] Failed to execute goal on project discovery-server: Could not resolve dependencies for project 
app.metatron.discovery:discovery-server:jar:2021.3-SNAPSHOT: Failed to collect dependencies at org.apache.hive:hive-
jdbc:jar:standalone:1.2.1000.2.5.5.0-157: Failed to read artifact descriptor for org.apache.hive:hive-
jdbc:jar:standalone:1.2.1000.2.5.5.0-157: Could not transfer artifact org.apache.hive:hive-jdbc:pom:1.2.1000.2.5.5.0-157 from/to 
maven-default-http-blocker (http://0.0.0.0/): Blocked mirror for repositories: [hortonworks-maven2-release-repository 
(http://repo.hortonworks.com/content/repositories/releases/, default, releases+snapshots), hortonworks-groups-public-repo 
(http://repo.hortonworks.com/content/groups/public, default, releases+snapshots)] -> [Help 1]
```

**To Reproduce**
Steps to reproduce the behavior:
1. git clone [master version]
2. mvn clean install -DskipTests

**Expected behavior**
Build fail

--- a/pom.xml
+++ b/pom.xml
@@ -807,13 +807,13 @@
         <repository>
             <id>hortonworks-maven2-release-repository</id>
             <name>Hortonworks Maven release repository</name>
-            <url>http://repo.hortonworks.com/content/repositories/releases/</url>
+            <url>https://repo.hortonworks.com/content/repositories/releases/</url>
         </repository>
 
         <repository>
             <id>hortonworks-groups-public-repo</id>
             <name>Hortonworks Groups Public Repository</name>
-            <url>http://repo.hortonworks.com/content/groups/public</url>
+            <url>https://repo.hortonworks.com/content/groups/public</url>
         </repository>
 
         <repository>

